app/controllers/booking_requests_controller.rb

```diff
@@ -1,32 +1,46 @@
 class BookingRequestsController < ApplicationController
   def index
+    @search = BookingRequestsSearchForm.new(search_params)
+    @search.status ||= BookingRequest.statuses.first
```

Comment:
I to and fro'd on this and settled on here because ultimately it's for the purpose of the interface that it needs to default to active and not blank/all.

Happy to move it into the form object though as agreed, it's neater there.